packages/server/graphql/public/typeDefs/_legacy.graphql

```diff
@@ -6521,6 +6521,21 @@ type TeamPromptMeeting implements NewMeeting {
   """
   id: ID!
 
+  """
+  The meeting series id this meeting is associated with if the meeting is recurring
+  """
+  meetingSeriesId: ID
+
+  """
+  The meeting series this meeting is associated with if the meeting is recurring
+  """
+  meetingSeries: MeetingSeries
+
+  """
+  The timestamp the meeting is scheduled to end
```

Comment:
+1 add something like "is non-null if meetingSeriesId is non-null" (if that's true!, I think it is?)